great_expectations/experimental/datasources/pandas_google_cloud_storage_datasource.py

```diff
@@ -51,7 +51,7 @@
     service_account = None
 
 
-class PandasGoogleCloudStorageError(PandasDatasourceError):
+class PandasGoogleCloudStorageDatasourceError(PandasDatasourceError):
```

Comment:
`non-blocking`
This seems a little overly specific.
Does this error really have anything to do with `Pandas`, seems like it's more of a `GoogleCloud` Error?

Reply:
@Kilo59  We followed the style of defining errors in each PandasDatasource subclass.  I see that there may be a better way -- in general, I prefer that all exceptions are defined in `great_expectations.exceptions` -- let us discuss on the best way forward.  Thanks.

Reply:
All of these error classes can be eliminated, they are not referenced anywhere

Reply:
this may have been left-over from before data connectors were added

Reply:
I think it's better for Errors to be defined next to where they are used and raised. I also think a generic DatasourceError would be more useful and easier to maintain than all these separate ones, especially since none of them capture and specific parameters. I'll create a ticket to clean them up